_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1994,12 +1999,14 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition(mfxExecuteParams *pParams)
         if (pParams->bComposite)
         {
             m_pipelineParam[refIdx].num_filters  = 0;
-#if defined(LINUX_TARGET_PLATFORM_BXT) || defined(LINUX_TARGET_PLATFORM_BXTMIN)
-            m_pipelineParam[refIdx].pipeline_flags |= VA_PROC_PIPELINE_SUBPICTURES;
-            m_pipelineParam[refIdx].filter_flags   |= VA_FILTER_SCALING_HQ;
-#else
-            m_pipelineParam[refIdx].pipeline_flags  |= VA_PROC_PIPELINE_FAST;
-#endif
+            if(MFX_HW_APL == hwType) {
```

Comment:
@IlyaSergeyev, make APL variant of pipeline properties default for all platforms

Reply:
@IlyaSergeyev : I am not sure why this is resolved? I still see APL check...